CHANGELOG.md

```diff
@@ -283,6 +283,10 @@ This can be accessed via the new Lua (R/W) `SettingsMan` property `AIUpdateInter
 
 - Added `LuaMan` Lua functions `GetDirectoryList(pathToGetDirectoryNamesIn)` and `GetFileList(pathToGetFileNamesIn)`, that get the names of all directories or files at the specified file path.
 
+- Added a new Lua event function for `Actor`s, `OnSelect(self, previousMode, previousPlayer)` that triggers when an Actors controller state changes (between AI/Player/Network control etc).
+This provides a script hook that fire when a player starts/stops controlling an actor.
```

Comment:
1. Actors controller state should be `Actor`'s `Controller` state. Similarly the other mention of actor at the end of the 2nd line should be `Actor`.

2. I dunno if you need to explain the arguments, when you can just name them as your explanations, i.e. `OnSelect(self, previousControllerMode, previousControllingPlayer)`.

3. If you want new lines without a bullet point, you need 2 spaces at the end

4. You should tab in the new lines (and maybe combine them)

Reply:
Fixed. Was missing an extension so my Markdown preview didn't match GitHub's.